device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceClient.java

```diff
@@ -698,18 +698,19 @@ public void startDeviceTwin(IotHubEventCallback deviceTwinStatusCallback, Object
      * @param deviceTwinStatusCallback the IotHubEventCallback callback for providing the status of Device Twin operations. Cannot be {@code null}.
      * @param deviceTwinStatusCallbackContext the context to be passed to the status callback. Can be {@code null}.
      * @param genericPropertiesCallBack the TwinPropertyCallBack callback for providing any changes in desired properties. Cannot be {@code null}.
-     * @param genericPropertyCallBackContext the context to be passed to the property callback. Can be {@code null}.     *
+     * @param genericPropertyCallBackContext the context to be passed to the property callback. Can be {@code null}.
      *
      * @throws IllegalArgumentException if the callback is {@code null}
      * @throws UnsupportedOperationException if called more than once on the same device
      * @throws IOException if called when client is not opened
      */
     public void startDeviceTwin(IotHubEventCallback deviceTwinStatusCallback, Object deviceTwinStatusCallbackContext,
-                                TwinPropertiesCallBack genericPropertiesCallBack, Object genericPropertyCallBackContext)
+                                TwinPropertiesCallback genericPropertiesCallBack, Object genericPropertyCallBackContext)
```

Comment:
I renamed your new interface ```TwinPropertiesCallBack``` to ```TwinPropertiesCallback``` since callback is one word. The original ```TwinPropertyCallBack``` unfortunately can't be changed to match, but we'll make do.